game/core/src/me/lihq/game/living/controller/PlayerController.java

```diff
@@ -45,6 +45,18 @@ public boolean keyDown(int keycode)
             return true;
         }
 
+        if (keycode == Input.Keys.J)
```

Comment:
Already done something like that, but I made it the F3 key, which would you prefer? I can easily change